### PR TITLE
Added conda install to readme

### PR DESCRIPTION
**Pull request by mpiseno**
## Motivation and Context

Updated readme with conda install instructions

### CODE REVIEW

**Comment by mpiseno** [comment]
Looks like the build documentation tests are failing even though the only file I changed is the README. Any pointers on this?

**Comment by aclegg3** [comment]
> Looks like the build documentation tests are failing even though the only file I changed is the README. Any pointers on this?

CI is dying in the same place for #528. 🤔 I don't think it is related to your changes.

File: README.md [hunk 1]
```diff
@@ -144,6 +144,22 @@ To run the above benchmarks on your machine, see instructions in the [examples](
 
 We provide a pre-built docker container for habitat-api and habitat-sim, refer to [habitat-docker-setup](https://github.com/facebookresearch/habitat-api#docker-setup).
 
+### Install with Conda
+
+We provide pre-built conda binaries for convenient installation, however please note that these might not be the most up to date versions of habitat sim.
```
[the remark this answers is not in the document]
Comment (dhruvbatra) [reply README.md:149]: Agreed. Is there a way for the CI to build a binary and upload to conda hub?
(One for every commit might be too much; maybe one binary for every release?)

File: README.md [hunk 2]
```diff
@@ -144,6 +144,22 @@ To run the above benchmarks on your machine, see instructions in the [examples](
...
+
+For systems with a display:
```
[the remark this answers is not in the document]
Comment (mpiseno) [reply README.md:151]: looking at lines 31-35 in conda-build/linux-matrix-builder.py, looks like I only built a headless linux version, but the infrastructure is there for other combinations of builds.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mpiseno) [reply README.md:149]: Makes sense. So should I explicity state which version/builds are available in the readme or should we hold off on this until a more permanent solution for automatic/scheduled builds is determined?

[the remark this answers is not in the document]
Comment (erikwijmans) [reply README.md:149]: Looks like we can schedule builds that run on master at some set interval: https://circleci.com/docs/2.0/workflows/#nightly-example

We could probably do something similar with the stable tag.

**Comment by dhruvbatra** [comment]
Closing because addressed in #789.

**Pull request closed without merging**